Networking/Networking/Model/TaxRate.swift

```diff
@@ -16,14 +16,14 @@ public struct TaxRate: Decodable, Equatable, GeneratedFakeable, GeneratedCopiabl
     ///
     public let name: String
 
-    /// Tax rate state.
-    ///
-    public let state: String
-
     /// Tax rate country.
     ///
     public let country: String
 
+    /// Tax rate state.
+    ///
+    public let state: String
```

Comment:
Previously, the property order doesn't match the initializer and thus some extra diffs from Copiable/Fakeable are required after running `rake generate` (this probably wasn't run when the changes were last made). With this change, the property order now matches the order in the initializer.

Reply:
cc @toupper about keeping the order of the properties and in the initializer consistent; @Ecarrion for the changes on `TaxRate`

Reply:
Thanks for tagging and letting me know 👍